understanding/20/use-of-color.html

```diff
@@ -157,18 +168,27 @@ <h2>Examples of Use of Color</h2>
       <dl>
          <dt>A form that uses color and text to indicate required fields</dt>
          <dd>A form contains both required and optional fields. Instructions at the top of the
-               form explain that required fields are labeled with red text and also with an icon.
-               Users who cannot perceive the difference between the optional field labels and the
-               red labels for the required fields will still be able to see the icon next to the
-               red labels.</dd>
+             form explain that required fields are labeled with red text and also with an icon.
+             Users who cannot perceive the difference between the optional field labels and the
+             red labels for the required fields will still be able to see the icon next to the
+             red labels.</dd>             
          <dt>An examination</dt>
          <dd>Students view an SVG image of a chemical compound and identify the chemical elements
-               present based <strong>both</strong> on the colors used, as well as numbers next to each
-               element. A legend shows the color and number for each type of element. Sighted users who
-               cannot perceive all the color differences can still understand the image by relying on
-               the numbers.</dd>
+             present based <strong>both</strong> on the colors used, as well as numbers next to each
+             element. A legend shows the color and number for each type of element. Sighted users who
+             cannot perceive all the color differences can still understand the image by relying on
+             the numbers.</dd>
+         <dt>A series of toggle buttons</dt>
+         <dd>Users are presented with a series of toggle buttons that can be turned on or off. Visually,
+             when a toggle button is turned on, it not only has a different background color, but is
+             also styled to appear "pressed", with a distinctive inner shadow. Alternatively, toggles that
```

Comment:
I suggest substituting the term "shadow" with an alternative. The word "shadow" may evoke associations with CSS features like box-shadow, which might not always be visible, especially in high contrast modes.

Reply:
as the current thinking is that HCM is still not directly covered by requirements of the SC (as it's effectively a customisation/change to author styles that is not standardised, so can be unpredictable in some cases for authors), I may leave this as is for now.